RELEASE.md

```diff
@@ -0,0 +1,92 @@
+# Boxo Releases
+
+## Release Policies
+### Breaking Changes
+TLDR: APIs may break between changes.
+
+There are two kinds of breaking changes to consider:
+
+1. **API refactors/changes**. There is equivalent functionality, but the API has changed. Generally adapting to these changes is trivial, and release notes will contain information about how to adapt.
+
+1. **Removal of a module or functionality**. In this case, Boxo maintainers will deprecate the relevant types in at least one release prior to their removal, and users will be forewarned in release notes. We strive to remove functionality as rarely as possible.
```

Comment:
```suggestion
1. **Removal of a module or functionality**. In this case, Boxo maintainers will deprecate the relevant types in at least one release prior to their removal, and users will be forewarned in release notes. This will not be done casually and justification will be provided.
```

You don't have to take this suggestion, but I expect we'll remove functionality more than "rarely" implies.  I want to make sure we have freedom/flexibility to stay trim/fit.